app/controllers/dashboards_controller.rb

```diff
@@ -48,7 +48,7 @@ def own_issues
 
   def former_issues(groups)
     return [] if groups.blank?
-    Issue.not_archived.includes(category: :main_category).where(
+    Issue.not_archived.authorized.includes(category: :main_category).where(
```

Comment:
```suggestion
    Issue.not_archived.includes(category: :main_category).where(
```
`former_issues` sind die ehemaligen Meldungen. Hier darf explizit nicht das `authorized` verwendet werden, da sonst die Liste immer leer ist, weil der Nutzer ja aktuell nicht mehr für diese Meldungen zuständig ist.
Da dieses nun mind. schon das 2. Mal der Fall eintritt, das an dieser Stelle versehentlich das `authorized` ergänzt wurde, bitte für diese Methode einen Test ergänzen mit Fixtures die eine Meldung enthalten die dem entprechenden Testnutzer einmal zugewiesen waren, aber aktuell nicht mehr sind. Um den Fall durch automatische Tests abzusichern!